specification/marketplacecatalog/data-plane/Search/stable/2022-02-02/search.json

```diff
@@ -0,0 +1,1065 @@
+          },
+          {
+            "type": "boolean",
+            "name": "isPowerBICertified",
+            "in": "query",
+            "x-nullable": true
+          },
+          {
+            "type": "boolean",
+            "name": "isIndustryCloud",
+            "in": "query",
+            "x-nullable": true
+          },
+          {
+            "type": "array",
+            "name": "operatingSystems",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "appSourceIndustries",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "appSourceCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azureIndustries",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azureCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azurePortalCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "string",
+            "name": "x-ms-app",
+            "x-originalName": "callingApp",
+            "in": "header",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "x-originalName": "apiVersion",
+            "in": "query",
+            "required": true,
+            "default": "2022-02-02",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a list of complete terms based on partial search term",
+            "schema": {
+              "$ref": "#/definitions/AutoCompleteResponseDto"
+            }
+          },
+          "default": {
+            "x-nullable": false,
+            "description": "*** Error Responses: ***\n\n * 400 - In case request malformed.\n\n * 500 - In case internal error.",
+            "x-ms-error-response": true,
```

Comment:
You don't need to specify `x-ms-error-response: true` on the `default` response.

Reply:
done